python/src/scipp/plot/sciplot.py

```diff
@@ -181,9 +213,17 @@ def _process_axes_dimensions(self,
             raise RuntimeError("Duplicate entry in axes: {}".format(self.axes))
 
     def savefig(self, filename=None):
+        """
+        Save plot to file.
+        Possible file extensions are `.jpg`, `.png` and `.pdf`.
+        """
         self.view.savefig(filename=filename)
 
     def as_static(self, keep_widgets=False):
+        """
+        Convert the plot to a static plot, releasing the memory held (a full
```

Comment:
A question for my sake. Is it possible to specify a plot should be static on creation and avoid making the memory copy at all?

Reply:
Hmm nice idea.
Currently no, the copy is always made, but you can do `plot(da).as_static()` to release it right away. But if you are very close to your limit, and you make a new plot, you could crash your kernel as you would temporarily exceed your capacity.

As an aside, I noticed that inside a notebook, when you create a 2d image and then convert it to static, you can still use the matplotlib to zoom and pan on the image, it just won't do the dynamic resample.